src/main/groovy/io/seqera/wave/service/aws/AwsMailProvider.groovy

```diff
@@ -54,15 +59,21 @@ class AwsMailProvider implements MailProvider {
     @Override
     void send(MimeMessage message, Mailer mailer) {
         //get mail client
-        final client = AmazonSimpleEmailServiceClientBuilder
-                .standard()
+        final client = SesClient.builder()
+                .region(Region.of(region))
                 .build()
```

Comment:
is it possible to \*not* specify the region?

Reply:
This is what I found in docs, if region is not provided then:

    1. Check the 'aws.region' system property for the region.
    2. Check the 'AWS_REGION' environment variable for the region.
    3. Check the {user.home}/.aws/credentials and {user.home}/.aws/config files for the region.
    4. If running in EC2, check the EC2 metadata service for the region.

Reply:
we have specified the region in bootstrap.yml, so we can remove it from here
https://github.com/seqeralabs/wave/blob/f8cfdcc17aa2f88cf12fc408f9c3e14c5ed47b78/src/main/resources/bootstrap.yml#L5

Reply:
But I think in the case of wave enterprise, we need to provide the option to the user, if the ses is not configured in the same region as their other stuff

Reply:
It can be assumed that the `AWS_REGION` is going to be available, that's the norm for EKS deployment

Reply:
ok sure, I will remove it